presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -285,6 +329,33 @@ public int getRunningQueries()
         }
     }
 
+    public int getAllRunningQueries()
+    {
+        synchronized (root) {
+            int allRunningQueries = getRunningQueries();
+            if (getPrev().isPresent()) {
+                allRunningQueries += getPrev().get().getRunningQueries();
+            }
+            return allRunningQueries;
+        }
+    }
+
+    public int getAllEligibleSubGroups()
```

Comment:
Private?

Reply:
can't do that as it's being used in tests.

Reply:
updated